scripts/deploy_assisted_service.sh

```diff
@@ -126,6 +126,13 @@ else
         DEBUG_DEPLOY_AI_PARAMS="REPLICAS_COUNT=1"
         # Override the SERVICE environment variable with the local registry debug image
         export SERVICE="${SUBSYSTEM_LOCAL_REGISTRY}/assisted-service:latest"
+
+        # Force removing the debugable image before re-deploying the service
+        # Change replicas to 0 so when applying the deployment it will identify the changes and allow deleting the
+        # image if it's already in use in the current pods
+        kubectl scale -n assisted-installer --replicas=0 deployment assisted-service || true
+        sleep 10
```

Comment:
can we leverage ``kubectl wait --for=delete`` instead of sleeping?

Reply:
Good idea. Thanks :)